fakes/bar.go

```diff
@@ -0,0 +1,17 @@
+package fakes
```

Comment:
Automating them with `counterfeiter` is lower maintenance and more in line with how we test with fakes elsewhere. I wouldn't worry about the size; it's not like they end up in the production code.